tests/makeotfexe_data/input/spec/3-2.fea

```diff
@@ -1 +1 @@
-include ( 2gi-1.fea )
+include ( 2gi-1.fea ) ;
```

Comment:
Is the semicolon required now?

Reply:
Yes, semicolons are required after include statements.

Reply:
(The addition of this test to the `spec` directory wasn't meant to have any strong implication. It's not clear that 8.a requires it. `makeotfexe` apparently does, though, so it needed a test.)

Reply:
It is a breaking change and needs a good justification (and documentation). It is a known fact that `makeotfexe` [allows `include` statements without terminating semicolon](https://github.com/adobe-type-tools/afdko/issues/114), and other implementations already [replicate this behavior](https://github.com/fonttools/fonttools/pull/642).

Reply:
This has been part of a deprecation message for a while now …

Reply:
`except` statement has been deprecated for more than a decade and it is still supported in this iteration.

Reply:
@khaledhosny see https://github.com/adobe-type-tools/afdko/pull/1367#discussion_r646953030